src/js/bootstrap-slider.js

```diff
@@ -741,9 +741,21 @@ const windowIsDefined = (typeof window === "object");
 			this.touchmove = this._touchmove.bind(this);
 
 			if (this.touchCapable) {
+				// Test for passive event support
+				var supportsPassive = false;
+				try {
+					var opts = Object.defineProperty({}, 'passive', {
```

Comment:
could you declare `opts` with `let` instead of `var` to ensure it is only scoped within the `try {}` block?